### ISSUE TITLE
Tuxedo Cat Chess x64 cannot be launched

### ISSUE THREAD

**Issue by ChessGUI** [nfmccrina/tuxedocat#1]
Hello, cf. my thread on CCC forum here:

http://talkchess.com/forum/viewtopic.php?topic_view=threads&p=697903&t=62402

Could you please build a Windows binary that does not require MS Visual Studio 2015 C++ or can run
with an older edition than MS Visual Studio 2015 C++ ?

**Comment by nfmccrina** [comment]
Sorry for the late response. Looks like you were able to figure out a work-around for the .dll issue. I don't have any computers running Windows at the moment, but I am installing some VMs and I will experiment with installs that don't have Visual Studio to see how I can prevent this from happening in the future.

**Comment by Mk-Chan** [comment]
You can try a static compile with the linux mingw-w64

**Comment by nfmccrina** [comment]
I did a static compile with Visual Studio in a VM. The binary was able to run in a clean (no development libraries installed) Windows 7 VM so I think it is good to go. I'm attaching these binaries here and I've also added them to the 0.02 release in the "releases" section. I am going to close this issue but feel free to contact me again if those don't work. Also, thank you for your time reporting this!

[TuxedoCat-0.02_x86.zip](https://github.com/nfmccrina/tuxedocat/files/674829/TuxedoCat-0.02_x86.zip)
[TuxedoCat-0.02_x86-64.zip](https://github.com/nfmccrina/tuxedocat/files/674830/TuxedoCat-0.02_x86-64.zip)
